cmake/SSGCommon.cmake

```diff
@@ -1251,6 +1251,6 @@ macro(ssg_build_vendor_zipfile ZIPNAME)
         redhat-zipfile
         DEPENDS "${CMAKE_BINARY_DIR}/vendor-zipfile/${ZIPNAME}-RedHat.zip"
     )
-    add_custom_target(vendor-zipfile ALL)
+    add_custom_target(vendor-zipfile)
```

Comment:
I was wondering why my build was always generating the vendor-zipfile when triggering ALL build. ;P